### ISSUE TITLE
Removing process separation between euphotic and deep ocean regions in iHAMOCC?

### ISSUE THREAD

**Issue by jmaerz** [NorESMhub/BLOM#399]
Hi @JorgSchwinger, @TomasTorsvik and @tjiputra ,
with #367, the option to remove the (artifical) separation between euphotic (`kwrbioz`) and deeper ocean region processes was granted. Moving forward to NorESMK2.5, I wonder and as suggested in #340, in how far we should aim at making this the new default and/or removing the current switch that we currently use to achieve this. Any thoughts?

**Comment by JorgSchwinger** [comment]
I have not looked in depth at any results with the option enabled. I would like to see a long model run first. My first impression was that the changes were large, so difficult to say at this stage.

**Comment by jmaerz** [comment]
I agree, it would be good to better understand the implications (in both, isopynic and hybrid coordinates). I suspect it thus makes sense to re-check the output and leave the switch in until a later stage.

**Comment by JorgSchwinger** [comment]
@jmaerz @TomasTorsvik 

I have now a 200 year model run with `lkrwbiozoff=.true.` (extended N-cycle turned off, hybrid coordinates). The results look terrible, but I think for most things I would know why and what needs to be changed. The primary thing is that the export from the euphotic zone is too small (surface nutrients increasing, and that the CaCO3 production needs to be tuned down substantially, since we have now a lot of recycled PP and "recycled export". Diagnostics is here:

https://ns2345k.web.sigma2.no/datalake/diagnostics/noresm/jschwing/NOINYOC_T62_tn21_51/HAMOCC_DIAG/yrs191to200-obs/index.html

Time series: (tn21_27=CMIP6, tn21_51=euphotic zone new, tn21_48=euphotic zone old, tn48_50=MA4GO)

[NorDiag_gts_T62_tn21_27-tn21_51-tn21_48-tn21_50_0001-0200.pdf](https://github.com/user-attachments/files/18800120/NorDiag_gts_T62_tn21_27-tn21_51-tn21_48-tn21_50_0001-0200.pdf)

**Comment by jmaerz** [comment]
HI @JorgSchwinger , thanks for providing this info. Just to be sure: tn21_51 is the only simulation, where you set `lkwrbioz_off=.true.`? I see that `wmin` is rather low, which likely explains part of the high surface nutrients.

**Comment by JorgSchwinger** [comment]
Yes, that is the only one. I'll try to increase `wmin`but also `epsher`is a knob to play with.

**Comment by jmaerz** [comment]
Closed by PR #574.
